website/cue/reference/releases/0.45.0.cue

```diff
@@ -0,0 +1,359 @@
+package metadata
+
+Ï: "0.45.0": {
+	date:     "2025-02-20"
+	codename: ""
+
+	whats_next: []
+
+	description: """
+		The Vector team is pleased to announce version 0.45.0!
+
+		Be sure to check out the [upgrade guide](/highlights/2025-02-24-0-45-0-upgrade-guide) for
+		breaking changes in this release.
+
+		This release contains a few notable new features along with the numerous enhancements and fixes as listed below:
+
+		- A new type of `enrichment_table`, `memory`, was introduced! This table can also act
+		  can also act as a sink! This enables new interesting use cases, for instance this table
+		  can be used as a cache or as an interface with an external key value store.
+		- A new `websocket_server` sink that acts as a websocket server and broadcasts events to all connected clients rather than only the first connected client.
```

Comment:
```suggestion
		- A new `websocket_server` sink that acts as a websocket server and broadcasts events to all connected clients, rather than just the first connected client.
```